src/Plugin/Field/FieldWidget/HelfiLinkitWidget.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\helfi_platform_config\Plugin\Field\FieldWidget;
+
+use Drupal\Component\Utility\UrlHelper;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Field\Attribute\FieldWidget;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Session\AccountProxyInterface;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+use Drupal\linkit\Plugin\Field\FieldWidget\LinkitWidget;
+use Drupal\linkit\Utility\LinkitHelper;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Overrides the linkit widget.
+ */
+#[FieldWidget(
+  id: "helfi_linkit",
+  label: new TranslatableMarkup('Helfi: Linkit'),
+  field_types: ['link']
+)]
+
+final class HelfiLinkitWidget extends LinkitWidget {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
```

Comment:
Missing return type (`self`).  

I think we should construct this without overriding the parent `__construct()`, so our implementation doesn't break if the parent class changes:

```
$instance = parent::create($container, $configuration, $plugin_id, $plugin_definition);
$instance->requestStack = $container->get(RequestStack::class);
return $instance;
```